java/code/src/com/suse/manager/webui/utils/InputValidator.java

```diff
@@ -33,8 +33,8 @@
      */
     INSTANCE;
 
-    // Allow letters (of all languages), numbers, '.', '/', '\' and '-'
-    private static final Pattern USERNAME = Pattern.compile("^[\\p{L}\\p{N}.-/\\\\]*$");
```

Comment:
@hustodemon so... After reviewing the tests (didn't notice we have them for this), I realized you added that maybe for future Windows support:
```
    public void testValidateBootstrapInputUserBackslash() {
        String json = "{user: 'domain\\\\admin', host: 'host.domain.com'}";
        BootstrapHostsJson input = MinionsAPI.GSON.fromJson(json, BootstrapHostsJson.class);
        List<String> validationErrors = InputValidator.INSTANCE.validateBootstrapInput(input);
        assertTrue(validationErrors.isEmpty());
    }
```
But that explains the backslash, not the slash. Any idea if that's used at Windows as well? I'd say it is not.

Reply:
After some searching, I wasn't able to verify the slash case on windows. But I'd say that the fact that it causes trouble on the supported systems by us (like SLE) is more than good reason to kick it away.